[2022-08-13 11:40:55,581] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_Raízen.create_raw_data_dir manual__2022-08-13T11:40:54.205193+00:00 [queued]>
[2022-08-13 11:40:55,603] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_Raízen.create_raw_data_dir manual__2022-08-13T11:40:54.205193+00:00 [queued]>
[2022-08-13 11:40:55,604] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-08-13 11:40:55,605] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-08-13 11:40:55,605] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-08-13 11:40:55,637] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): create_raw_data_dir> on 2022-08-13 11:40:54.205193+00:00
[2022-08-13 11:40:55,644] {standard_task_runner.py:52} INFO - Started process 1481 to run task
[2022-08-13 11:40:55,650] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_Raízen', 'create_raw_data_dir', 'manual__2022-08-13T11:40:54.205193+00:00', '--job-id', '319', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Raizen.py', '--cfg-path', '/tmp/tmpe8yj_7jt', '--error-file', '/tmp/tmpc7l57eut']
[2022-08-13 11:40:55,651] {standard_task_runner.py:80} INFO - Job 319: Subtask create_raw_data_dir
[2022-08-13 11:40:55,745] {task_command.py:369} INFO - Running <TaskInstance: ETL_Raízen.create_raw_data_dir manual__2022-08-13T11:40:54.205193+00:00 [running]> on host 4bbb34367c41
[2022-08-13 11:40:55,835] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ETL_Raízen
AIRFLOW_CTX_TASK_ID=create_raw_data_dir
AIRFLOW_CTX_EXECUTION_DATE=2022-08-13T11:40:54.205193+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-13T11:40:54.205193+00:00
[2022-08-13 11:40:55,837] {ETL_Raizen.py:27} INFO - Dir already exists
[2022-08-13 11:40:55,838] {python.py:173} INFO - Done. Returned value was: None
[2022-08-13 11:40:55,859] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ETL_Raízen, task_id=create_raw_data_dir, execution_date=20220813T114054, start_date=20220813T114055, end_date=20220813T114055
[2022-08-13 11:40:55,904] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-13 11:40:55,952] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
